kokoro/gcp_ubuntu/bazel/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2019 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
Don't bump copyright years, in general. They should be the oldest applicable year. So basically once you've created a file, you never touch the year again

Reply:
Gotcha! This was an artifact of the renames.